apps/ledger-live-mobile/src/screens/SignTransaction/02-ConnectDevice.tsx

```diff
@@ -57,16 +77,8 @@ function ConnectDevice({
       />
       <DeviceAction
         action={action}
-        request={{
-          account,
-          parentAccount,
-          appName,
-          transaction,
-          status,
-          tokenCurrency,
-          dependencies,
-          requireLatestFirmware: true,
-        }}
+        // @ts-expect-error Wrong types?
+        request={request}
```

Comment:
It looks like it's because `transaction` could be `undefined` or `null` while it should not for the `request` type -> but i don't understand why it was not complaining before ?

Reply:
Me neither, and since this is a hot fix, I dont want to understand it now.
It's actually the same on LLD, bunch of expect errors until we figure it out

Reply:
Yep agree !